extensions/azure-eventhubs/runtime/src/main/java/io/quarkiverse/azure/eventhubs/runtime/EventhubsConfig.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkiverse.azure.eventhubs.runtime;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigMapping(prefix = "quarkus.azure.eventhubs")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface EventhubsConfig {
+
+    /**
+     * The flag to enable the eventhubs. If set to false, the eventhubs will be disabled
+     */
+    @WithDefault("true")
+    boolean enabled();
+
+    /**
+     * The fully qualified namespace of Azure Event Hubs. Required if quarkus.azure.eventhubs.enabled is set to true
+     */
```

Comment:
Looking at https://github.com/quarkiverse/quarkus-azure-services/pull/310/files#diff-e705484273a930a7c489016dce68bb5469f27907a7edf5d052c2f6ad1b49be44R74, it requires short name of the namespace, instead of a fully qualified namespace, need to be consistent.

Reply:
Fixed